4.3/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if [[ -n "$ADMINER_DESIGN" ]]; then
+    ln -s "designs/$ADMINER_DESIGN/adminer.css" .
```

Comment:
I'd say: with the permission change on top `ln` in the container will always work. If someone maps a drive into the container and still has `ADMINER_DESIGN` set, it should throw an error to notify the user about the problem.